rtloader/test/datadog_agent/datadog_agent.go

```diff
@@ -2,6 +2,8 @@ package testdatadogagent
 
 import (
 	"fmt"
+	traceconfig "github.com/DataDog/datadog-agent/pkg/trace/config"
+	"github.com/mailru/easyjson/jlexer"
```

Comment:
These need to follow import conventions (stdlib block, TP block, same-project imports)